EvidenceApi.Tests/IntegrationTests.cs

```diff
@@ -52,6 +52,7 @@ public void BaseSetup()
             MockNotifyClient = CreateMockNotifyClient();
             _factory = new MockWebApplicationFactory<TStartup>(_connection, MockNotifyClient.Object, Options);
             Client = _factory.CreateClient();
+            Client.DefaultRequestHeaders.Add("UserEmail", "resident-dummy-value");
```

Comment:
Let's add a comment here to mention why we added it? Maybe something like: _setting value for UserEmail HTTP header so BaseController passes the request to the controller_

Reply:
👍